.pre-commit-config.yaml

```diff
@@ -0,0 +1,5 @@
+- repo: https://github.com/jumanjihouse/pre-commit-hooks
+  rev: master  # or specific git tag
```

Comment:
Seems like we might want to use a specific version here to defend against unexpected changes.